pkg/cmd/admin/policy/reconcile_sccs.go

```diff
@@ -244,8 +244,8 @@ func (o *ReconcileSCCOptions) computeUpdatedSCC(expected kapi.SecurityContextCon
 	sort.StringSlice(expected.Users).Sort()
 
 	// make a copy of the expected scc here so we can ignore metadata diffs.
-	updated := expected
 	expected.ObjectMeta = actual.ObjectMeta
+	updated := expected
```

Comment:
This will fail with resource version conflicts. Need to specifically copy labels and annotations, I'm pretty sure